refinery/data_set_manager/utils.py

```diff
@@ -9,22 +9,19 @@
 import time
 import urlparse
 
+import core
```

Comment:
I think `known_first_party` would do it, but we'd have to list all the modules we wanted handled that way. Significant maintenance cost / little benefit. (Also: I have got it to complain about syntax errors in the config file, but I haven't actually seen it make any difference: I'm doing something wrong, and it feels like another thing that might break in the future.)